### ISSUE TITLE
Make the interface of ETmoses something like ETmodel

### ISSUE THREAD

**Issue by grdw** [quintel/etmoses#780]
This is the current layout:

![current](https://cloud.githubusercontent.com/assets/2676542/13001602/b617c42a-d167-11e5-9ca9-5524aec65daf.png)

This leaves lots of whitespace near the edges. Which makes this a really sad screenshot:

![screen shot 2016-02-12 at 9 05 43 am](https://cloud.githubusercontent.com/assets/2676542/13001624/db796250-d167-11e5-8394-0472b41f7bc0.png)

The chart on the middle-right is really really cramped while there's lots of space at the left and right.

I would prefer to change the layout to something like ETmodel:

![current-new](https://cloud.githubusercontent.com/assets/2676542/13001638/fbf930d2-d167-11e5-8186-a281de1dc51c.png)

This is just a really quick drawing I made so don't take it to seriously - it's just to get the idea accross. The idea was to create something similar to Gitlab:

![screen shot 2016-02-12 at 9 08 06 am](https://cloud.githubusercontent.com/assets/2676542/13001657/32df4c4e-d168-11e5-979d-f70a557ac92e.png)

Another reason for this change is - when there's going to be more charts in the future (which I guess it will) it will only become more and more of a hassle to fit them into the small stroke in the middle.

Including @ChaelKruip @antw @dennisschoenmakers

**Comment by ChaelKruip** [comment]
I like it that you are looking into this @grdw!

Also, the technology 'matrix' could use a big re-design IMO. Is this part of this effort as well?

**Comment by grdw** [comment]
> Also, the technology 'matrix' could use a big re-design IMO. Is this part of this effort as well?

Not for now but this could be like a kickstarter for that; at least there will be more 'space' available to make nicer designs. :smile:

**Comment by antw** [comment]
I'm not sure I follow why moving the menu to the left would make *more* space for charts? [Most people are not using maximised 27" displays](http://gs.statcounter.com/#resolution-ww-monthly-201412-201505). I *am*, but my browser window is [just wide enough](https://cloud.githubusercontent.com/assets/4383/13003157/c9c2de66-d16b-11e5-8574-45357ec0f041.png) to accommodate the Moses and ETModel UIs (that screenshot also has roughly the same whitespace as the site does when maximised on a 15" 1800p HiDPI display).

I've always found the left-aligned menu on ETModel to be quite limiting, because it reduces available horizontal screen space which could be dedicated to sliders or charts. But it's there by necessity as there are far too many items for a top-aligned menu.

**Comment by grdw** [comment]
Let's go back to the origin of the problem than. I understand the concern @antw is sketching.

![screen shot 2016-02-12 at 10 51 09 am](https://cloud.githubusercontent.com/assets/2676542/13003577/9e1ce42c-d176-11e5-8943-54994ecc27bb.png)

I find this to be a not so optimal use of the 'chart'. It just looks really cramped in a tiny box. I know it was my original idea to set them side by side but right now I think it just looks bad. My original thought would be that if I make use of all the whitespace *I see on my wide screen* (which is where the thought went south), it wouldn't look so cramped.

Come to think of it - I get a better idea. Use tabs to split the view into two. This way you can show really large topologies properly and the chart will be wide like on the "load_profiles/show" page.

![design etmoses-one](https://cloud.githubusercontent.com/assets/2676542/13003898/60422e76-d178-11e5-8c9e-e53697ad2f4b.png)

![design etmoses-two](https://cloud.githubusercontent.com/assets/2676542/13003897/6041f5e6-d178-11e5-831a-f2c6747a735d.png)

** Ovbiously here 'Households 1' needed to be selected in the tab menu but again - for the sake of the idea.

**Comment by antw** [comment]
> I find this to be a not so optimal use of the 'chart'. It just looks really cramped in a tiny box.

I agree, and having the load chart the full width of the page would be really nice. I'm definitely with you there. :laughing: 

What about if we reduced the topology diagram to be like the simple version on the import form (no labels, nodes closer together, show labels when hovering the node) and show the full-size topology in a separate tab as suggested, or when pressing a "maximise" button?

I think the topology needs to be visible *somewhere* on the load chart page, otherwise viewing the loads of different nodes will become a frustrating experience of constantly switching tabs.

**Comment by grdw** [comment]
Some Firefox HTML hackerdiehackie shows my perference:

![screen shot 2016-02-12 at 11 25 57 am](https://cloud.githubusercontent.com/assets/2676542/13004361/69174664-d17b-11e5-8d49-0f26da2443c8.png)

I would love it if the first thing people see when they arrive on the show page is this. When they click on a node it will appear as a new tab and you see the chart "full width" (not 3D yet ... ).
